src/app/admin/auditLogTable/AuditLogTable.tsx

```diff
@@ -26,6 +27,8 @@ const AuditLogTable: React.FC = () => {
     const [currentPage, setCurrentPage] = useState(1);
     const startPoint = (currentPage - 1) * auditLogCountPerPage;
     const endPoint = currentPage * auditLogCountPerPage - 1;
+    const [modalIsOpen, setModalIsOpen] = useState<boolean>(false);
```

Comment:
Do we really need this boolean state? Can't we use the presence of `selectedAuditLog` as a way to signal whether to show the modal?

Reply:
That's much better yes! I'm wondering if, within the modal component, there's a way for typescript to know that selectedAuditLog must be truthy?

Reply:
![image](https://github.com/NorwoodAndBrixtonFoodbank/nbf-website/assets/138508884/c98cbae2-6b0a-4fbd-bf24-a82d979605b1)
ie remove the need for the {selectedAuditLog && ....} check

Reply:
you could do 
```ts
if (selectedAuditLogRow === null) {
  return <></>
}
```
or something although this is also not super clean / not aligned with the `isOpen` way of showing modals.

Up to you.

Reply:
I'll leave it how it is